src/archivematicaCommon/lib/archivesspace/client.py

```diff
@@ -118,6 +118,26 @@ def _delete(self, url, params={}, expected_response=200):
                              params=params,
                              expected_response=expected_response)
 
+    def _format_notes(self, record):
+        """
+        Extracts notes from a record and reformats them in a simplified format.
+        """
+        if len(record['notes']) == 0:
```

Comment:
Records are guaranteed to contain a `notes` property. Could check truthiness though.